src/renderer.rs

```diff
@@ -509,7 +584,10 @@ pub fn render_error(
                         }
                     }
                     @if !conf.hide_version_footer {
-                        (version_footer())
+                        p.footer {
```

Comment:
I'm not sure whether we should always display the wget command. It might be more annoying than just the footer and perhaps those should not universally be hidden together. What do you think about making showing the download command opt-in behind a flag?

Reply:
Yup can we do